model/lifecycle.go

```diff
@@ -1644,3 +1627,25 @@ func addNewTasks(ctx context.Context, batchTimeInfo batchTimeTasksAndVariants, v
 
 	return taskIds, nil
 }
+
+func getTaskIdTables(v *Version, p *Project, newPairs TaskVariantPairs) (TaskIdConfig, error) {
+	if utility.StringSliceContains(evergreen.AllTaskRequesterTypes, v.Requester) {
```

Comment:
Is this meant to include anything for which `patch_optional` doesn't apply? If that's true then git tag versions should also be in this if statement. (Could probably use `!IsPatchRequester(v.Requester)`)